configuration_helper/README.rst

```diff
@@ -0,0 +1,111 @@
+====================
+Configuration Helper
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-tools/tree/11.0/configuration_helper
+    :alt: OCA/server-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-tools-11-0/server-tools-11-0-configuration_helper
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/149/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+*This module is intended for developer only. It does nothing used alone.*
+
+It helps to create `config.settings` by providing an abstract Class.
+
+This class:
+
+  * creates automatically related fields in 'whatiwant.config.settings'
+    using those defined in 'res.company': it avoids duplicated field definitions.
+  * company_id field with default value is created
+  * onchange_company_id is defined to update all related fields
+  * supported fields: char, text, integer, float, datetime, date, boolean, m2o
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+.. code-block:: python
+
+    from . company import ResCompany
+
+    class WhatiwantClassSettings(orm.TransientModel):
+        _inherit = ['res.config.settings', 'abstract.config.settings']
+        _name = 'whatiwant.config.settings'
```

Comment:
Probably more explicit if you could replace `whatiwant` by `res` everywhere because no more several configs file

Reply:
@bealdav updated

Reply:
Thanks but
updated the wrong file ;-) It must be in readme/ folder

Reply:
oups :) updated!